x-pack/plugins/search_playground/server/routes.ts

```diff
@@ -90,76 +90,82 @@ export function defineRoutes({
     errorHandler(async (context, request, response) => {
       const [, { actions }] = await getStartServices();
       const { client } = (await context.core).elasticsearch;
-      try {
```

Comment:
Why did you move code out of try/catch block? 
For example now if error appears during JSON.parse of query_fields we don't see any error, right? I thought we agreed to show that type of messages in system error messages list.

Reply:
`try catch` should not that big, We want to make sure adding this block for specific purposes only. 

`JSON.parse of query_fields` is an internal error and we log that error for debugging purpose. For most cases, It does not require any action from End user so showing this specific error will not be helpful to them.

Reply:
Also this route is using the `errorHandler()` helper which is already doing a `try/catch` on the entire route. so adding another `try/catch` around all the values is redundant. We should use one of the other. 

IMO the `errorHandler()` should be used for unexpected errors and we should be deliberate about using `try/catch` within the route/lib for specific potential errors.

Reply:
Yeah, sorry I mean to handle errors in errorHandler to return proper response to when something is unexpectedly broken

Reply:
Yeah removing try catch was a right thing to do